bot/orchestration/src/main/kotlin/ai/tock/bot/orchestration/shared/secondaryBotAction.kt

```diff
@@ -38,9 +38,9 @@ interface SecondaryBotAction {
     fun toAction(metaData: OrchestrationMetaData) : Action
 
     companion object {
-        fun from(action : Action) : SecondaryBotAction? = when (action) {
-            is SendSentence -> SecondaryBotSendSentence(messages = action.messages.filterIsInstance<SerializableConnectorMessage>(), text = action.text?.toString(), metadata = action.metadata)
-            is SendChoice -> SecondaryBotSendChoice(intentName = action.intentName, parameters = action.parameters, metadata = action.metadata)
+        fun from(action: Action, botId: String) : SecondaryBotAction? = when (action) {
```

Comment:
Action, si l'interface SecondaryBotAction est utilisée autre part que dans tock, ça casse la retrocompatibilité

Reply:
Thanks for alerting. Yes, there will be a compatibility break. 
Normally, it should not be used outside tock code but if it is, it is better to force clients to add bot id to call.